Apps/AdminWebClient/src/Server/Startup.cs

```diff
@@ -161,7 +162,6 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                         endpoints.MapToVueCliProxy(
                             "{*path}",
                             new SpaOptions { SourcePath = "ClientApp" },
-                            "serve",
```

Comment:
What was this change for?

Reply:
I didn't realize that had happened until I was looking at this PR. I think it was a code cleanup thing. I looked at it and the default value for that item is "serve" so I think the code cleanup just removed it.